spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -89,21 +90,25 @@ def get_parser():
         default=1)
     misc.add_argument(
         "-largest",
+        dest='keep_largest',
         type=int,
         help="Keep the largest connected-objects from the output segmentation. Specify the number of objects to keep."
              "To keep all objects, set to 0",
-        default=0)
+        default=None)
     misc.add_argument(
         "-fill-holes",
```

Comment:
```suggestion
        "-fill-holes",
        dest="fill_holes",
```

Maybe it is not needed?

Reply:
By default, the destination value is `fill_holes` (since the flag name is `-fill-holes`), so it is not required